### PR TITLE
[docs] Add route description and summary to remaining routes

### PR DESCRIPTION
**Pull request by fe51**
This PR aims at adding summary and descriptions to the remaining routes (accesses, users, devices, installations and alerts)

image
image
image
image
image

Any feedback is welcome !

### CODE REVIEW

**Review by florianriche — DISMISSED** [review]

File: src/app/api/routes/alerts.py [hunk 1]
```diff
@@ -27,30 +33,45 @@ async def create_alert_from_device(payload: AlertBase,
...
-async def delete_alert(alert_id: int = Path(..., gt=0)):
+async def delete_alert(alert_id: int = Path(..., gt=0), summary="Delete a specific alert"):
+    """
+    Based on a alert_id, deletes the specified alert
+    """
     return await crud.delete_entry(alerts, alert_id)
 
 
-@router.put("/{alert_id}/link-media", response_model=AlertOut)
+@router.put("/{alert_id}/link-media", response_model=AlertOut, summary="Link a alert to a media")
```
Comment (frgfm) [inline src/app/api/routes/alerts.py:68]: "a alert" --> "an alert"

**Review by frgfm — CHANGES_REQUESTED** [review]
Thanks for the PR! Just a small typo, the rest is all good!

File: src/app/api/routes/alerts.py (hunk 1 above)
Comment (fe51) [reply src/app/api/routes/alerts.py]: Nice catch ! thanks

**Review by frgfm — APPROVED** [review]
Thanks for the edit! Only need to merge master now ;)

File: src/app/api/routes/installations.py [hunk 2]
```diff
@@ -53,8 +53,12 @@ async def delete_installation(installation_id: int = Path(..., gt=0)):
     return await crud.delete_entry(installations, installation_id)
 
 
+<<<<<<< HEAD
 @router.post("/site-devices/{site_id}", response_model=List[int],
              summary="Get all devices related to a specific site")
+=======
+@router.get("/site-devices/{site_id}", response_model=List[int])
+>>>>>>> master
```
Comment (frgfm) [inline src/app/api/routes/installations.py]: From master, you only need to keep the post -> get edit :+1:

**Review by frgfm — COMMENTED** [review]
My apologies, but there is a conflict to fix, and you need to merge master again since it was just updated on one of the files you edited here :upside_down_face: 

Feel free to add summary and description on the new device route :ok_hand:

File: src/app/api/routes/installations.py (hunk 2 above)
Comment (fe51) [reply src/app/api/routes/installations.py]: oops, I did not save my work so my previous commit was useless.  I think we are good now

**Review by frgfm — APPROVED** [review]
Thanks again! (I merged master myself and resolved conflicts)

**Pull request merged**